include/SFML/System/Time.hpp

```diff
@@ -37,7 +37,7 @@ namespace sf
 /// \brief Represents a time value
 ///
 ////////////////////////////////////////////////////////////
-class SFML_SYSTEM_API Time
+class Time
```

Comment:
Doesn't the class still need to be exported?

Reply:
No, it's an inline class now. It's the same as in #1904.